### PR TITLE
cadenceworkflow.io site overhaul

### PR DESCRIPTION
**Pull request by mfateev**
Added 

- concepts section
- use cases section

Still needs to redo the front page links.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -15,8 +15,14 @@
...
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "https://cadenceworkflow.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+#TODO: remove temporary overrides
+#baseurl: "" # the subpath of your site, e.g. /blog
+#url: "https://cadenceworkflow.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+baseurl: "/cadence" # the subpath of your site, e.g. /blog
+url: "https://mfateev.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+
```
Comment (dmetzgar) [inline _config.yml]: This will need to be restored before merging

File: docs/03_concepts/05_queries.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Workflow Query
+
+Worfklow fault-oblivious
```
Comment (dmetzgar) [inline docs/03_concepts/05_queries.md]: It seems like the content is missing from this page

File: index.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ callout_description: Develop resilient long-running business applications with s
   </div>
 
   <div class="feature timers">
-    <a href="{{ '/docs/02_key_features#durable-timers' | relative_url }}">
+    <a href="{{ '/docs/>03_key_features#durable-timers' | relative_url }}">
```
Comment (dmetzgar) [inline index.md]: What's the added `>` for?

**Comment by dmetzgar** [comment]
`08_cli` and `09_archival` also show up in your docs site. I think maybe you didn't stage the delete of these files? They should be a rename since this PR adds `07_cli` and `08_archival`.

**Comment by dmetzgar** [comment]
What I think we should do is once this is ready, just merge. Then have @purplesusan do her edits directly instead of putting comments. That will save some time.

File: docs/03_concepts/01_workflows.md [hunk 4]
```diff
@@ -52,62 +52,61 @@ public class SubscriptionWorkflowImpl implements SubscriptionWorkflow {
...
 
-Cadence guarantees that there could be only one workflow (across all workflow types) with given ID open per domain at any time. An attempt to start workflow with the same ID is going to fail with `WorkflowExecutionAlreadyStarted` error.
+Cadence guarantees that there could be only one workflow (across all workflow types) with a given ID open per domain at any time. An attempt to start a workflow with the same ID is going to fail with `WorkflowExecutionAlreadyStarted` error.
 
 An attempt to start a workflow if there is a completed workflow with the same ID depends on a `WorkflowIdReusePolicy` option:
 
 - `AllowDuplicateFailedOnly` means that it is allowed to start a workflow only if a previously executed workflow with the same ID failed.
 - `AllowDuplicate` means that it is allowed to start independently of the previous workflow completion status.
-- `RejectDuplicate` mans that it is not allowed start a workflow execution using the same workflow ID at all.
+- `RejectDuplicate` means that it is not allowed start a workflow execution using the same workflow ID at all.
```
Comment (dmetzgar) [inline docs/03_concepts/01_workflows.md]: not allowed *to* start

**Comment by dmetzgar** [comment]
Hey @purplesusan, can you sign the CLA? On the conversation tab of the PR, go to the bottom. There should be a "Review Requested" box with a link to "Show all checks". Click that and then click on Details next to license/cla. It will take a few seconds for the CLA to load.

**Comment by purplesusan** [comment]
@dmetzgar sorry, I thought I did this the other day, but I guess it didn't complete. It's done now.

File: docs/100_contact_us.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Contact Us
+
+If you have a question, check if it is already answered at stackoverflow.com under [cadence-workflow](https://stackoverflow.com/questions/tagged/cadence-workflow) tag.
```
Comment (purplesusan) [inline docs/100_contact_us.md]: check if it is already answered at stackoverflow.com -> check whether it is already answered at stack overflow

File: docs/100_contact_us.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+If you still need help get it through the [Cadence Slack Workspace](https://join.slack.com/t/uber-cadence/shared_invite/enQtNDczNTgxMjYxNDEzLTQyYjcxZDM2YTIxMTZkMzQ0NjgxYmI3OWY5ODhiOTliM2I5MzA4NTM4MjU4YzgzZDkwNGEzOTUzNTBlNDk3Yjc).
```
Comment (purplesusan) [inline docs/100_contact_us.md]: help get it through the -> help, visit

**Pull request merged**